storage/src/mmr/journaled.rs

```diff
@@ -343,13 +343,13 @@ impl<E: RStorage + Clock + Metrics, H: Hasher> Mmr<E, H> {
         for pos in Proof::<H>::nodes_to_pin(prune_to_pos) {
             let digest = self.get_node(pos).await?.unwrap();
             self.metadata.put(
-                U64::new(Self::NODE_PREFIX, pos),
+                U64::new(NODE_PREFIX, pos),
```

Comment:
Thankfully we've seen a use for metadata not just being a `u8`. Without this proof point, I probably would've overly simplified this.